website/pages/docs/auth/aws.mdx

```diff
@@ -573,6 +573,28 @@ Policy](#recommended-vault-iam-policy) except it doesn't need any further
 Furthermore, in the master account, Vault must be granted the action `sts:AssumeRole`
 for the IAM Role to be assumed.
 
+### AWS Instance Metadata Timeouts
+
+~> Affects Vault 1.4 and later
+
+Anytime Vault needs to connect to the [EC2 instance metadata
+service][aws-ec2-mds] on an EC2 instance, there may be a delay. The AWS SDK used
+by Vault first attempts to connect to [v2 of the instance metadata service
+(IMDSv2)][aws-ec2-imdsv2], and if that times out, it falls back to v1. In Vault
+1.4, this timeout could take up to 2 minutes. In Vault 1.5 and later, it can
+take up to 2 seconds. The timeout occurs in situations such as Vault running in
```

Comment:
We tested specifically for the Docker case, but I think [this can also happen](https://github.com/aws/aws-sdk-go/issues/2989#issuecomment-562381634) if the host has a proxy in front of it which requires an extra hop before it can reach IMDSv2.

Reply:
Good point, I reworded that part a bit to make it more general, and mention docker as a specific case.